src/helpers/Helpers.ts

```diff
@@ -0,0 +1,21 @@
+export function getNumbers(from: number, to: number): number[] {
```

Comment:
this file name shouldn't start with a capital letter if there are only helper functions (no Component). Fair for all such files